src/qt/addresstablemodel.cpp

```diff
@@ -88,7 +89,8 @@ class AddressTablePriv
                         QString::fromStdString(address.purpose), address.is_mine);
                 cachedAddressTable.append(AddressTableEntry(addressType,
                                   QString::fromStdString(address.name),
-                                  QString::fromStdString(EncodeDestination(address.dest))));
+                                  QString::fromStdString(EncodeDestination(address.dest)),
+                                  (address.dest.type() == typeid(PKHash))));
```

Comment:
6604abe6ab25925b379b3f16630906de3aaf51fb

I think you could do `std::get_if<PKHash>(address.dest) != nullptr`. Same below, and drop <typeinfo> header.

nit, commit prefix should be `qt`?

Reply:
There's [std::holds_alternative](https://en.cppreference.com/w/cpp/utility/variant/holds_alternative) for that.